crates/cext/cbindgen.toml

```diff
@@ -72,6 +72,7 @@ prefix = "Qk"
 "SparseObservable" = "Obs"
 "CSparseTerm" = "ObsTerm"
 "CircuitData" = "Circuit"
+"DAGCircuit" = "Dag"
```

Comment:
Just to bikeshed on names here, I think this is more natural as `QkDAG` or `QkDAGCircuit`. I think I prefer the `QkDAG` name. Since we use camel case for the type names `Dag` doesn't really work because "dag" is really an acronym for 3 words not a single word.

Reply:
There's different schools of thought on acronyms in camel case - Python's PEP8 says all caps, but [Rust's sAPI guide](https://rust-lang.github.io/api-guidelines/naming.html) says to pretend it's a word. When I prototyped, `QkDag` just looked better to me so I went with that.

I'm against `DAGCircuit` for the function prefix because it's unnecessarily long, but I can live with it for the type if we really want.

Reply:
I can't live with `DAGCircuit`, personally.

As for `QkDAG`, my habit would be to leave it as `QkDag` since I learned C# in my formative years after uni, which by convention only capitalizes the letters of acronyms with 2 letters lol:

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

Reply:
As long as it's `qk_dag_*` for functions I don't care all that much about the type. I just loosely prefer "treat acronyms as words" for camel case because I find it easier to read.

Let's take a team vote between `QkDag` and `QkDAG` for the C naming convention then call it settled.